factories/admin.rb

```diff
@@ -26,6 +26,10 @@
     factory :open_doors_admin do
       roles { ["open_doors"] }
     end
+
+    factory :official_admin do
+      roles { ["official"] }
+    end
```

Comment:
Can you remove this, since we don't have an official role for admins? (Also, my apologies for hitting the single comment button rather than start a review. :facepalm:)